lib/opal/nodes/iter.rb

```diff
@@ -1,52 +1,19 @@
 # frozen_string_literal: true
 
-require 'opal/nodes/node_with_args'
+require 'opal/nodes/def'
```

Comment:
Not sure why this change, looks like we're still inheriting from NodeWithArgs, is there another reason?

Reply:
My mistake. I wanted to do one thing inheriting another.